resource_customizations/argoproj.io/AnalysisRun/health.lua

```diff
@@ -0,0 +1,32 @@
+hs = {}
+if obj.status ~= nil then
+    if obj.status.status == "Pending" then
+        hs.status = "Progressing"
+        hs.message = "Waiting for analysis run to finish: status is pending."
+    end
+    if obj.status.status == "Running" then
+        hs.status = "Progressing"
+        hs.message = "Waiting for analysis run to finish: status is running."
+    end
+    if obj.status.status == "Successful" then
+        hs.status = "Healthy"
+        hs.message = "Waiting for analysis run to finish: status is successful."
```

Comment:
The text, `Waiting for analysis run to finish` from here below is inaccurate. Can we reword to something like:

Analysis run completed successfully"
Analysis run failed
Analysis run had error
Analysis run was inconclusive

Reply:
fixed